staging/src/k8s.io/api/discovery/v1/types.go

```diff
@@ -165,12 +155,15 @@ type EndpointPort struct {
 	// This field follows standard Kubernetes label syntax.
 	// Un-prefixed names are reserved for IANA standard service names (as per
 	// RFC-6335 and http://www.iana.org/assignments/service-names).
-	// Non-standard protocols should use prefixed names.
-	// Default is empty string.
+	// Non-standard protocols should use prefixed names such as
+	// mycompany.com/my-custom-protocol.
```

Comment:
I  think this is something I missed porting back to v1alpha1 but matches what we [already have in v1beta1](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/discovery/v1beta1/types.go#L164-L165). The git diff is weird here because v1alpha1 is being removed at the same time v1 is being added, so it's treating everything as a rename (and catching changes I or others missed adding to v1alpha1).

Reply:
I would suggest removing v1alpha1 in a separate commit (or maybe even a separate PR)

Reply:
changing v1a1 -> v1 made this PR easy to review